### PR TITLE
Added Empty View in Notes Tab

### LINKED ISSUE #662: Add empty view for notes in video player

### PR DESCRIPTION
**Pull request by prathamk22**
Fixes #662 
Changes: Added a Linearlayout which contains empty view of notes icon, title and sub text of it

Screenshots for the change:
vlcsnap-2020-05-11-10h13m05s221

### CODE REVIEW

File: app/src/main/res/layout/fragment_notes.xml [hunk 1]
```diff
@@ -1,10 +1,48 @@
...
+            android:textAlignment="center"
+            android:textColor="@color/orangish"
+            android:textSize="18sp" />
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/margin_extra_small"
+            android:fontFamily="@font/gilroy_bold"
+            android:text="Click on + to add a new note."
```
Comment (aggarwalpulkit596) [inline app/src/main/res/layout/fragment_notes.xml]: Move all hardcoded strings to strings.xml

File: app/src/main/java/com/codingblocks/cbonlineapp/mycourse/player/notes/VideoNotesFragment.kt [hunk 2]
```diff
@@ -86,6 +93,16 @@ class VideoNotesFragment : BaseCBFragment(), AnkoLogger {
         }
     }
 
+    private fun showEmptyView(show: Boolean) {
+        if (show) {
+            playerNotesRv.visibility = View.VISIBLE
```
Comment (aggarwalpulkit596) [inline app/src/main/java/com/codingblocks/cbonlineapp/mycourse/player/notes/VideoNotesFragment.kt]: use isVisible = true or isVisible = false

File: app/src/main/java/com/codingblocks/cbonlineapp/mycourse/player/notes/VideoNotesFragment.kt [hunk 3]
```diff
@@ -76,7 +76,14 @@ class VideoNotesFragment : BaseCBFragment(), AnkoLogger {
         playerNotesRv.setRv(requireContext(), notesListAdapter, false)
 
         viewModel.notes.observer(viewLifecycleOwner) {
-            notesListAdapter.submitList(it)
+            if (it.isNullOrEmpty()) {
```
Comment (aggarwalpulkit596) [inline app/src/main/java/com/codingblocks/cbonlineapp/mycourse/player/notes/VideoNotesFragment.kt]: can be simplified by
`showEnptyView(it.isNullOrEmpty())`

**Review by aggarwalpulkit596 — CHANGES_REQUESTED** [review]

**Comment by prathamk22** [comment]
@aggarwalpulkit596 changes done as per feedback.

**Review by aggarwalpulkit596 — APPROVED** [review]

**Pull request merged**